ecc/molec/ECC-MOLEC-000001.yml

```
@@ -0,0 +1,70 @@
+state: proposed
+name: H3 K27M Mutation
```

```suggestion
name: H3 K27M alteration
```

I submit we should center on the name ("<description>" alteration) for point mutations.

---

I'm not going to suggest changes for every instances of mutation -> alteration, but please make those (and use alteration in the future).

---

Thanks for the suggestions, I'd like to note that **alteration** term is a board term in genetics definition, which can be refer to various type of DNA changes including stucture variations, deletions, insertions, and more; not just a point mutation. Personally, I think that **mutation**" or "**mutant** would be more percise in the context. These terms are commonly used in scientific papers. That said, I also understood that we might want to generalize our terminolgy, so let's discuss futher to find the best convention for us to avoid ambiguity.

---

I lean towards mutation when the variant is limited to SNV and/or Indel such as in this case.  When the type of hit can be more broad, then alteration.